rails/spec/controllers/api/v1/permission_forms_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+RSpec.describe API::V1::PermissionFormsController, type: :controller do
+  describe '#index' do
+    it 'GET index' do
+      skip "Skipping this test for now"
```

Comment:
You should probably go ahead and add the spec test before it gets left unimplemented.